### DISCUSSION TITLE
Floating-ui with Lit

### DISCUSSION THREAD

**Discussion by mc2** [floating-ui/floating-ui#2685] [Ideas]
Hi, working in TypeScript and Lit.dev. Need to refactor a dropdown to stay inside viewport and came across floating-ui.
See React and Vue versions. Can there be a Lit version?
Thanks!

**Comment by atomiks** [comment]
There can, but no one who maintains the lib has experience with Lit :-)

`@floating-ui/dom` is simple enough to wrap in your framework of choice for the most part as it's conceptually simple
